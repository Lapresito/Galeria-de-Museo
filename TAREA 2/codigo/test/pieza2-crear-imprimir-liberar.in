crearFecha 1/1/2001
crearPieza 7 pieza1 Carlos Nelson
idPieza
imprimirPieza
liberarPieza
crearFecha 2/7/2003
crearPieza 3 pieza2 Luciana Diaz
imprimirPieza
idPieza
liberarPieza
Fin
