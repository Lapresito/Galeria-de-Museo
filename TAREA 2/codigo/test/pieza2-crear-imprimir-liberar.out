Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Fecha creada en forma exitosa.
2> Pieza creada en forma exitosa.
3> ID: 7
4> Pieza: 7
pieza1-Carlos Nelson
Creada en: 1/1/2001
5> Pieza liberada en forma exitosa.
6> Fecha creada en forma exitosa.
7> Pieza creada en forma exitosa.
8> Pieza: 3
pieza2-Luciana Diaz
Creada en: 2/7/2003
9> ID: 3
10> Pieza liberada en forma exitosa.
11> Fin.
